sweepbatcher/sweep_batcher_test.go

```diff
@@ -48,15 +47,22 @@ var destAddr = func() btcutil.Address {
 	return addr
 }()
 
-var senderKey, receiverKey [33]byte
+var htlcKeys = func() loopdb.HtlcKeys {
```

Comment:
nit: instead of declaring as variable, you could just declare it as function.

Reply:
Then key generation would happen every time this variables is used (many times in this file). I want to cache the values.